sdp/processors/datasets/coraa/create_initial_manifest.py

```diff
@@ -0,0 +1,117 @@
+import glob
+import os
+from pathlib import Path
+from typing import List
+from huggingface_hub import snapshot_download
+import pandas as pd
+
+import rarfile  #Needs to be installed
+import sox
+from sox import Transformer
+
+from sdp.processors.base_processor import BaseParallelProcessor, DataEntry
+from sdp.utils.common import extract_archive
+
+class CreateInitialManifestCORAA(BaseParallelProcessor):
+    """
+        Processor to creatie initial manifest file fo CORAA ASR dataset
+
+         Dataset link: https://github.com/nilc-nlp/CORAA
+
+         Args:
+            raw_data_dir (str): the path to the directory in which all the data will be downloaded.
+            extract_archive_dir (str): directory where the extracted data will be saved.
+            data_split (str): "train", "dev" or "test".
+            already_extracted (bool): if True, we will not try to extract the raw data.
+                Defaults to False.
+            already_downloaded (bool): if True, we will not try to download files.
+            exclude_dataset: list: list of the dataset names that will be excluded when creating initial manifest.
```

Comment:
please add other args ("resampled_audio_dir", "target_samplerate", "target_nchannels", any others that are missing)

Reply:
Done